docs-ref-services/servicebus.md

```diff
@@ -351,5 +356,11 @@ sent_msg = Message(b'receive message',
 )
 ```
 
+## Next Steps
+* [Service Bus documentation](https://docs.microsoft.com/azure/service-bus-messaging)
+* [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus)
+* [SDK reference documentation](https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python)
+* [Additional samples](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus/examples)
+
 > [!div class="nextstepaction"]
 > [Explore the Management APIs](/python/api/overview/azure/servicebus/management)
```

Comment:
Recommend removing this button. I think it was originally added during an early pass at adding these SDK overview articles (aka "ref landing pages"), and since this particular article is all about the client package, the button's target is not really applicable.